tests/acceptance/stepDefinitions/filesContext.js

```diff
@@ -1128,7 +1128,13 @@ When('the user tries to move file/folder {string} into folder {string} using the
   return client.page.FilesPageElement.filesList().attemptToMoveResource(resource, target)
 })
 
-When('the user cancels an attempt to move file/folder {string} using the webUI', function (
+When('the user cancels an attempt to move file/folder {string} using the webUI', function(
+  resource
+) {
+  return client.page.FilesPageElement.filesList().cancelMoveResource(resource)
+})
+
+When('the user cancels an attempt to copy file/folder {string} using the webUI', function(
   resource
 ) {
   return client.page.FilesPageElement.filesList().cancelMoveResource(resource)
```

Comment:
```suggestion
  return client.page.FilesPageElement.filesList().cancelCopyResource(resource)
```
And create `cancelCopyResource` function in `filesList` page.